strategies/affected/src/test/java/org/arquillian/smart/testing/strategies/affected/fakeproject/test/ATest.java

```diff
@@ -0,0 +1,17 @@
+package org.arquillian.smart.testing.strategies.affected.fakeproject.test;
+
+import org.arquillian.smart.testing.strategies.affected.fakeproject.main.A;
+import org.junit.Ignore;
+import org.junit.Test;
+
+@Ignore
```

Comment:
Would be worth to add explanation to `@Ignore` - might help in the future to avoid confusion why we have such "tests"